server/build.gradle

```diff
@@ -15,6 +16,7 @@ plugins {
 }
 apply plugin: 'org.hibernate.orm'
 
+def jooqSrcDir = 'src/jooq/java'
```

Comment:
Is it a convention to use this folder name? Otherwise I'd name it `src/main/jooq-gen` or so.

Reply:
The default path is `/build/generated-src/jooq`. I can change it  to `src/main/jooq-gen`.